freqtrade/constants.py

```diff
@@ -191,6 +191,7 @@
                 "process_throttle_secs": {'type': 'integer', 'minimum': 600},
                 "calculate_since_number_of_days": {'type': 'integer'},
                 "allowed_risk": {'type': 'number'},
+                "capital_available_percentage": {'type': 'number'},
```

Comment:
which of the edge-properties are required if edge is on?

maybe add that to the "required" section within the edge dict ... ?

Reply:
good on 👍